### PR TITLE
Edit template to avoid some repetitions

### PR DESCRIPTION
**Pull request by apiraino**
Should fix https://github.com/rust-lang/compiler-team-prioritization/issues/10

not the most elegant solution, but I tried not to touch the templates too much.

Old text
```
- Pending FCP requests (check your boxes!)
  - No pending FCP requests on compiler-team repo this time.
  - No pending FCP requests on rust repo this time.
  - No pending FCP requests on forge repo this time.
```

new rendering
```
- Pending FCP requests (check your boxes!)
  - No pending FCP requests this time.
```

and equivalent for other FCP and MCP (cannot test everything until github returns nothing)

@spastorino how does it look?

### CODE REVIEW

**Comment by apiraino** [comment]
self note: when merged [this line](https://github.com/rust-lang/rust-forge/blame/master/src/compiler/prioritization/procedure.md#L117) can go away

**Comment by Mark-Simulacrum** [comment]
This is waiting on approval from @spastorino.

**Comment by spastorino** [comment]
This is not exactly how I'd have imagine fixing this problem. What I would have done is just have one call that returns the whole collection of issues. We would need to make a call to our logic that contacts all those three queries into just one list.

On the other hand, we can also merge this that fixes the issue and try to find a more elegant solution later.
So for now I'm :+1: with this :).

**Review by spastorino — APPROVED** [review]

**Comment by spastorino** [comment]
Having a second thought now ... @apiraino does this break new lines separations and such in some way?.

**Comment by apiraino** [comment]
> I would have done is just have one call that returns the whole collection of issues

@spastorino That's the approach I had discarded but I'm ok going that way and work on something more invasive but more structured, not a big deal.

> Having a second thought now ... @apiraino does this break new lines separations and such in some way?.

I've tested with this weeks agenda (2020-08-27) and didnt notice anything weird happening when rendering. Pleae let me know if something slipped my checks.

**Comment by spastorino** [comment]
Let's merge this as it solves the existing issue and we can work on a better solution afterwards. cc @Mark-Simulacrum

**Comment by spastorino** [comment]
We would need to revert this as it doesn't really work. @apiraino is already aware of it. We should go directly to the real solution :).

**Pull request merged**
